bionic/code_hasher.py

```diff
@@ -249,12 +274,25 @@ def _update_hash(self, hash_accumulator, obj, code_context):
             self._update_hash_for_code(hash_accumulator, obj, code_context)
 
         elif inspect.isclass(obj):
-            # TODO: Hashing classes is next on our roadmap. Let's hash
-            # the name for now.
+            add_to_hash(hash_accumulator, type_prefix=TypePrefix.CLASS)
```

Comment:
Will we do this for classes defined in libraries, like `pandas.DataFrame`? Should we?

Reply:
I don't think it's too bad to hash them if we are whitelisting hashable properties. Any of the more complex types like methods will be filtered out in the `inspect.isroutine` condition.

But since we are now considering blacklisting non-hashable properties instead, it might be worth skipping classes like we do routines. I'll try it when I'm addressing the [other comment](https://github.com/square/bionic/pull/319#discussion_r549373846).

Reply:
Done!